.maestro/browser_features/search_bar.yaml

```diff
@@ -0,0 +1,56 @@
+# search_bar.yaml
+appId: com.duckduckgo.mobile.ios
+tags:
+    - release
```

Comment:
The initial plan was to keep tags aligned to folders to make it clearer what was running at any point. It looks like we've already lost this over time, so this is likely fine at this point.